### DISCUSSION TITLE
Course Review | Summer

### DISCUSSION THREAD

**Discussion by shashankn75** [snigi-gupta/AttendingUniversityAtBuffalo#29] [Q&A] [answered]
Hi Snigdha,

I would need review on a few Summer courses.

CSE510 - Malware Analysis -by  Dr. Justin Del Vecchio  - I have contacted the prof, and as for the course layout he is yet to plan the layout for summer21.  I wanted to know how the course load would be, since the course is a 6-week schedule.

CSE542 - Software Engineering Concepts - by Matthew Hertz  - What kind of course load to expect in this subject, since it is a 6-week schedule with two 3-hrs lectures in a week. 

thank-you

**Comment by snigi-gupta** [comment, marked as answer]
Hi @shashankn75 
Firstly I apologize for the delay in my response. Lately, I have been dealing with a lot of change, hence the delay.

I didn't take both these courses, but I can definitely say that Summer courses are not that vigorous. They are doable and definitely a good way to make some credits during the summer in case you don't have an internship.

For 542, I have heard is that Matthew Herts is a good prof and not too strict with hard assignments. It is a medium course in my opionion.
